setup.py

```diff
@@ -1,9 +1,13 @@
 #!/usr/bin/env python
 from setuptools import setup, find_packages
 
+with open('viper/common/version.py') as f:
```

Comment:
I justed tested it with simply importing (`from viper.common.version import __version__ `) and it appears to be working just the same. 

The reason for the way I proposed it is this StackOverflow answer: http://stackoverflow.com/a/16084844/4623390. There it is indicated that when running setup.py it's better to not use an import.. have you experienced any issue with that in PyMISP? I just tried and didn't notice anything bad.